src/frontend/components/StudentLiveWrapper/StudentLiveViewerWrapper/index.spec.tsx

```diff
@@ -1,210 +0,0 @@
-      title: 'live title',
-      live_state: liveState.RUNNING,
-      urls: {
-        manifests: {
-          hls: 'https://marsha.education/live.m3u8',
-        },
-        mp4: {},
-        thumbnails: {},
-      },
-    });
-
-    render(
-      wrapInIntlProvider(
-        <StudentLiveViewerWrapper video={video} playerType="videojs" />,
-      ),
-    );
-
-    await waitFor(() =>
-      // The player is created
-      expect(mockCreatePlayer).toHaveBeenCalledWith(
-        'videojs',
-        expect.any(Element),
-        expect.anything(),
-        video,
-        'en',
-        expect.any(Function),
-      ),
-    );
-  });
-
-  it('renders the advertisment when live is idling', async () => {
-    const video = videoMockFactory({
-      title: 'live title',
-      live_state: liveState.IDLE,
-    });
-
-    render(
-      wrapInIntlProvider(
-        <StudentLiveViewerWrapper video={video} playerType="videojs" />,
-      ),
-    );
-
-    screen.getByRole('heading', {
-      name: 'Live is starting',
-    });
-  });
-
-  it('renders the advertisment when live is starting', async () => {
-    const video = videoMockFactory({
-      title: 'live title',
-      live_state: liveState.STARTING,
-    });
-
-    render(
-      wrapInIntlProvider(
-        <StudentLiveViewerWrapper video={video} playerType="videojs" />,
-      ),
-    );
-
-    screen.getByRole('heading', {
-      name: 'Live is starting',
-    });
-  });
-
-  it('renders the advertisment until manifest can be fetch', async () => {
-    fetchMock.mock('https://marsha.education/live.m3u8', 404);
-    const video = videoMockFactory({
-      title: 'live title',
-      live_state: liveState.RUNNING,
-      urls: {
-        manifests: {
-          hls: 'https://marsha.education/live.m3u8',
-        },
-        mp4: {},
-        thumbnails: {},
-      },
-    });
-
-    render(
-      wrapInIntlProvider(
-        <StudentLiveViewerWrapper video={video} playerType="videojs" />,
-      ),
-    );
-
-    await waitFor(() => {
-      expect(
-        fetchMock.calls('https://marsha.education/live.m3u8', {
-          method: 'GET',
-        }),
-      ).toHaveLength(1);
-    });
-
-    screen.getByRole('heading', {
-      name: 'Live is starting',
-    });
-
-    fetchMock.mock('https://marsha.education/live.m3u8', 200, {
-      overwriteRoutes: true,
-    });
-
-    jest.advanceTimersToNextTimer();
-
-    await waitFor(() => {
-      expect(
-        fetchMock.calls('https://marsha.education/live.m3u8', {
-          method: 'GET',
-        }),
-      ).toHaveLength(2);
-    });
-
-    expect(
-      screen.queryByRole('heading', {
-        name: 'Live is starting',
-      }),
-    ).not.toBeInTheDocument();
-    await waitFor(() =>
-      // The player is created
-      expect(mockCreatePlayer).toHaveBeenCalledWith(
-        'videojs',
-        expect.any(Element),
-        expect.anything(),
-        video,
-        'en',
-        expect.any(Function),
-      ),
-    );
-  });
-});
```

Comment:
Shouldn't we move those tests in another src/frontend/components/StudentLiveWrapper/index.spec.tsx ?

Reply:
it is already tested in `configures live state without action elements while the live is not really started` for a viewer and in `configures live state with action elements even if the live is not really started` for a student on stage (for a student on stage we are not waiting to pull the manifest, so this has no effect and we test jitsi is still called)